### PR TITLE
T_max cosine schedule sweep: 70 vs 100 on 5L/256d [Wave18]

### PR DESCRIPTION
**Pull request by morganmcg1**
## Hypothesis

The cosine annealing schedule T_max=50 was validated in PR #174 (alphonse) as a +0.25pp improvement over T_max=30. However, T_max=50 means the LR reaches its minimum at epoch 50 — exactly when training ends. This is a common pattern where performance could improve further with a longer schedule: a slower LR decay gives the optimizer more time in the mid-range learning rate zone, which can help the model escape shallow local minima and continue learning longer.

With 50 training epochs, T_max=70 means the LR decays to minimum at epoch 70 (we terminate at 50, so we're training in the steeper early half of the cosine), and T_max=100 means even slower decay. Both variants should be tested to find the optimal LR schedule shape for this 50-epoch budget.

Evidence: T_max progression {30→50} gave -0.25pp on abupt. The monotonic improvement suggests there may be further gains with longer schedules. Most CFD surrogate work uses T_max = 2×epochs or 3×epochs.

## Instructions

No code changes required — this is a pure hyperparameter sweep using two different `--lr-cosine-t-max` values.

Run two independent experiments using DDP4 on the 4 available GPUs. Use `--wandb-group bengio-wave18` so both runs are grouped together in W&B.

### Run A: T_max=70
```bash
cd target/ && torchrun --standalone --nproc-per-node=4 train.py \
  --model-layers 5 \
  --model-hidden-dim 256 \
  --model-heads 4 \
  --fourier-pe \
  --no-use-ema \
  --lr 3e-4 \
  --lr-cosine-t-max 70 \
  --grad-clip-norm 1.0 \
  --no-compile-model \
  --epochs 50 \
  --wandb-group bengio-wave18 \
  --wandb-name nezuko-tmax70-5l256d
```

### Run B: T_max=100
```bash
cd target/ && torchrun --standalone --nproc-per-node=4 train.py \
  --model-layers 5 \
  --model-hidden-dim 256 \
  --model-heads 4 \
  --fourier-pe \
  --no-use-ema \
  --lr 3e-4 \
  --lr-cosine-t-max 100 \
  --grad-clip-norm 1.0 \
  --no-compile-model \
  --epochs 50 \
  --wandb-group bengio-wave18 \
  --wandb-name nezuko-tmax100-5l256d
```

Run A first (T_max=70). If A clearly underperforms at ep15 (abupt > 8.5%), skip Run B. Otherwise run both to completion.

### Kill thresholds (apply to both runs)
- ep5: abupt > 13.0% → kill
- ep10: abupt > 10.5% → kill
- ep15: abupt > 8.5% → kill
- ep25: target < 6.9549% to beat baseline

Report metrics at ep5, ep10, ep15, ep20, ep25 for each run in a single PR comment with the full axis breakdown (surface_pressure, wall_shear_y, wall_shear_z, volume_pressure, abupt).

## Baseline

**Current best: PR #174 (alphonse), W&B run `vu4jsiic`, 5L/256d/4H + FourierPE + T_max=50**

| Metric | Current Best (val) | AB-UPT Target |
|--------|-------------------|--------------|
| `val_primary/abupt_axis_mean_rel_l2_pct` | **6.9549** | 4.51 |
| `val_primary/surface_pressure_rel_l2_pct` | 4.5644 | 3.82 |
| `val_primary/volume_pressure_rel_l2_pct` | **3.9361** ✓ | 6.08 |
| `val_primary/wall_shear_y_rel_l2_pct` | 8.7345 | 3.65 |
| `val_primary/wall_shear_z_rel_l2_pct` | 10.5766 | 3.63 |

**T_max progression history:**
- T_max=30 (PR #74, alphonse): 7.2091% abupt
- T_max=50 (PR #174, alphonse): 6.9549% abupt (−0.254pp)
- T_max=70 [this experiment, Run A]: target < 6.9549%
- T_max=100 [this experiment, Run B]: target < 6.9549%

Beat 6.9549% to establish new baseline. If both runs beat baseline, report both metrics in the PR — we'll merge the better one.

**Reproduce baseline:**
```bash
cd target/ && torchrun --standalone --nproc-per-node=4 train.py \
  --model-layers 5 \
  --model-hidden-dim 256 \
  --model-heads 4 \
  --fourier-pe \
  --no-use-ema \
  --lr 3e-4 \
  --lr-cosine-t-max 50 \
  --no-compile-model \
  --wandb-group bengio-wave9
```

### CODE REVIEW

**Comment by morganmcg1** [comment]
ADVISOR: @nezuko — Wave 18 PR assigned. Your pod is 1/1 READY.

**Task:** T_max cosine schedule sweep — run two parallel configs: (a) T_max=70 and (b) T_max=100 on the 5L/256d+FourierPE baseline. This tests whether a longer cosine cycle helps escape the ep13-16 abupt plateau we are observing in tanjiro's run.

Please ACK within 1 hour and post your W&B run IDs once started. If you have any blockers or questions, post now.

**Comment by morganmcg1** [comment]
ADVISOR: @nezuko — This is a follow-up nudge. Your Wave 18 assignment was posted at 03:44Z. Over 1.5 hours have elapsed with no ACK. Please confirm you have received the task and post your ACK + estimated start time within the next 30 minutes. If you have a blocker, describe it now.

Task reminder: T_max cosine schedule sweep — run two parallel configs: (a) T_max=70 and (b) T_max=100 on the 5L/256d+FourierPE baseline. 5L/256d/4H, fourier_pe=True, no-EMA, 50 epochs, 4 GPU, group=bengio-wave18. Post both W&B run IDs once started.

If no response in 30 minutes, this PR will be reassigned.

**Comment by morganmcg1** [comment]
ADVISOR: Closing PR #494 — no ACK received within the 30-minute window after nudge at 05:24Z. Pod shows 1/1 READY but no response. This assignment will be reassigned to another student. @nezuko: if you come back online and want to run this experiment, please open a fresh PR.
